### ISSUE TITLE
Focus order on filter sections incorrect

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon-platform#506]
from @mbgower 

> 
> BTW, that filter layout is a great example of a trivial but real glitch in [Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)
> If you look at the layout, I think we can agree that the order should be by column:
> Sponsor>Tags>Status>Platform>Framework.
> Instead, it’s by row: Sponsor>Status>Framework>Tags>Platform.
> I don’t think it represents a failure of any consequence, but it’s a good example of how the visual structure doesn’t match the implemented ux. 

![image](https://user-images.githubusercontent.com/2753488/162027100-adb7b8ce-4580-4ba9-b189-e62f9da70441.png)

**Comment by mbgower** [comment]
This is relevant to both reading and focus order, btw. Best way to solve is probably if the elements are implemented in the DOM in the order intended for reading and focus order, then CSS is used to visually enhance
